exercises/D-logical-operators/exercise.js

```diff
@@ -11,14 +11,14 @@ let cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-let htmlLevelAbove5;
-let cssLevelAbove5;
+let htmlLevelAbove5 = htmlLevel > 5 && htmlLevel < 15;
```

Comment:
Hi Saadia, thanks for your review. The use of the && was because I felt the question asked for the use of a logical operator, plus it was in the logical operators exercise. But I did realise it was unnecessary.